plugin-unit-tests.md

```diff
@@ -1,29 +1,40 @@
 # Plugin Unit Tests
 
-This guide will demonstrate how to:
+## Running tests on Travis CI
 
-* Set up unit tests for an existing plugin, using WP-CLI
-* Run the tests locally
+If you host your plugin on Github and enable [Travis CI](http://about.travis-ci.org), the tests will be run automatically after every commit you make to the plugin.
 
-We're going to assume that you already have a plugin called `my-plugin`.
+All you need to do to enable this is copy and then edit the following files from the [WP-CLI sample plugin](https://github.com/wp-cli/sample-plugin):
```

Comment:
> although to do this wouldn't you need to setup the entire environment locally?

Ah, yes, I suppose that's an implicit assumption (that you have a functioning local dev environment).